### PR TITLE
Refactor default page template

### PR DESCRIPTION
**Pull request by mattbowen-usds**
Since the default page and the lifecycle page share so much in common, I'm trying to DRY up the code so there's just really one parameterized template that other page types can configure.

### CODE REVIEW

File: tech-far-hub/content/resources/contract-solutions-vehicles/8a.mdx [hunk 1]
```diff
@@ -5,6 +5,8 @@ page_type: initiative
 template: default
 nav_weight: 10
 description: The U.S. Digital Service and Small Business Administration have partnered to help agencies buy digital services using this low-risk gateway.
+media_image: "/assets/img/shahadat-rahman-BfrQnKBulYQ-unsplash.jpg"
```
Comment (vim-usds) [inline tech-far-hub/content/resources/contract-solutions-vehicles/8a.mdx:8]: You'll have to open source this new CMS you're creating! Nice!

File: tech-far-hub/src/components/site-layout.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { ReactNode } from "react";
-import "@trussworks/react-uswds/lib/uswds.css";
-import "@trussworks/react-uswds/lib/index.css";
+import { Link } from "gatsby";
+//import "@trussworks/react-uswds/lib/index.css";
```
Comment (vim-usds) [inline tech-far-hub/src/components/site-layout.tsx:4]: I'm placing this in a global css file and it seems to work well.

File: tech-far-hub/src/pages/template-lifecycle.tsx [hunk 3]
```diff
@@ -1,98 +1,37 @@
...
   pageContext,
 }: LifecycleInnerPageProps) => {
-  const currentSlug = data.currentPage?.frontmatter?.slug;
-  const tocLinks = data.currentPage?.tableOfContents?.items.map((item: ITOCItem) => {
+  if (data.currentPage.frontmatter && data.siblings && data.currentPage.tableOfContents) {
     return (
-      <a href={item.url} key={item.url} className="font-ui-3xs">
-        {item.title}
-      </a>
+      <PageLayoutNav
```
Comment (vim-usds) [inline tech-far-hub/src/pages/template-lifecycle.tsx:20]: Talk about way cleaner!

**Review by vim-usds — APPROVED** [review]
Looking awesome!

File: tech-far-hub/src/components/site-layout.tsx (hunk 2 above)
Comment (mattbowen-usds) [reply tech-far-hub/src/components/site-layout.tsx]: Oh good catch --- I did not mean to just leave that sitting around.

**Pull request merged**